interfaces/builtin/bluez.go

```diff
@@ -196,13 +205,38 @@ func (iface *BluezInterface) PermanentPlugSnippet(plug *interfaces.Plug, securit
 
 func (iface *BluezInterface) ConnectedPlugSnippet(plug *interfaces.Plug, slot *interfaces.Slot, securitySystem interfaces.SecuritySystem) ([]byte, error) {
 	switch securitySystem {
-	case interfaces.SecurityDBus:
-		return bluezConnectedPlugDBus, nil
 	case interfaces.SecurityAppArmor:
-		return bluezConnectedPlugAppArmor, nil
+		old := []byte("@SLOT_SECURITY_TAGS@")
+		var new []byte
+		switch {
+		case len(slot.Apps) == 1:
+			for appName := range slot.Apps {
+				new = []byte(fmt.Sprintf("snap.%s.%s", slot.Snap.Name(), appName))
+			}
+		case len(slot.Apps) == len(slot.Snap.Apps):
+			new = []byte(fmt.Sprintf("snap.%s.*", slot.Snap.Name()))
+		case len(slot.Apps) != len(slot.Snap.Apps):
+			buf := bytes.NewBuffer(nil)
+			fmt.Fprintf(buf, "snap.%s.{", slot.Snap.Name())
+			appNames := make([]string, 0, len(slot.Apps))
+			for appName := range slot.Apps {
+				appNames = append(appNames, appName)
+			}
+			sort.Strings(appNames)
+			for i, appName := range appNames {
```

Comment:
We should also move this code somewhere else as I will otherwise copy&paste it to the networkmanager interface where we have to do the same thing.